### PR TITLE
feat(client): remove bootstrap Form component

### PR DESCRIPTION
**Pull request by Sboonny**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

I don't know why it doesn't provide styles change, or why its attribute doesn't change anything. 🤷

### CODE REVIEW

File: client/src/components/Donation/stripe-card-form.tsx [hunk 1]
```diff
@@ -136,7 +136,10 @@ const StripeCardForm = ({
   };
 
   return (
-    <Form className='donation-form' onSubmit={handleSubmit}>
+    <form
+      className='donation-form'
+      onSubmit={event => void handleSubmit(event)}
```
Comment (Sboonny) [inline client/src/components/Donation/stripe-card-form.tsx:141]: The other changes should be "affecting" the styles, except this one, it affects the functionality, as well

Although I think, that it is the same functionality, it doesn't hurt to check it.

**Comment by ahmaxed** [comment]
Then form on the update-email.tsx is slightly effected. 
We can adopt the same ui for the update-email.tsx 's from as for the email inputs in settings.

**Review by ahmaxed — APPROVED** [review]

File: client/src/components/settings/keyboard-shortcuts.tsx [hunk 2]
```diff
@@ -17,8 +16,7 @@ export default function KeyboardShortcutsSettings({
   const { t } = useTranslation();
 
   return (
-    <Form
-      inline={true}
+    <form
```
Comment (ojeytonwilliams) [inline client/src/components/settings/keyboard-shortcuts.tsx:19]: Are there any reasons to keep this form wrapper? a11y, maybe? It doesn't seem to be doing anything.

File: client/src/components/settings/sound.tsx [hunk 3]
```diff
@@ -40,7 +39,7 @@ export default function SoundSettings({
   }
 
   return (
-    <Form inline={true} onSubmit={(e: React.FormEvent) => e.preventDefault()}>
+    <form onSubmit={(e: React.FormEvent) => e.preventDefault()}>
```
Comment (ojeytonwilliams) [inline client/src/components/settings/sound.tsx:42]: Same again.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]

File: client/src/components/settings/keyboard-shortcuts.tsx (hunk 2 above)
Comment (ojeytonwilliams) [reply client/src/components/settings/keyboard-shortcuts.tsx:19]: Sure thing. Let's keep this focused!

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Pull request merged**